modules/@apostrophecms/piece-type/index.js

```diff
@@ -62,7 +62,6 @@ module.exports = {
             label: 'Both'
           }
         ],
-        allowedInChooser: false,
```

Comment:
This is leftover from 2.x. Not used in 3.0.

Reply:
I guess not? Choosing something in the trash is still a goofy thing to do.

Reply:
In fact... I think we should keep it, not delete it, but implementing it doesn't have to be a subject for this particular PR. Just leaving it in so it can be implemented in a separate ticket for frontend.

Reply:
Okay. We probably need a ticket to track getting this back in on the back end.

Reply:
I don't think it was ever on the back end. The front end knows when it's showing a chooser and when it's showing the regular manage modal, and it can use this flag appropriately.

Reply:
(It's not a security thing, these filters wouldn't do anything horrible, it's just bad UX to let someone "choose" something that is currently in the trash? Maybe? I guess you could use that feature to remove it from the trash before choosing it, but that's a whole UX convo.)